Automattic/wp-calypso
# Stats: Remove StatsFirstView component

#### Changes proposed in this Pull Request

* The `StatsFirstView` component is distracting/jarring, taking the user out of the flow when viewing their site management area for the first time. We don't use this pattern elsewhere in Calypso, and it doesn't seem to serve a purpose. We propose removing it as part of the Fixing the Flows initiative in an effort to simplify the dashboard.

**Before**

<img width="1280" alt="Screen Shot 2019-04-22 at 12 02 41 PM" src="https://user-images.githubusercontent.com/2124984/56510207-1dbaec80-64f7-11e9-9f61-975f6bf4e430.png">

**After**

<img width="1280" alt="Screen Shot 2019-04-22 at 12 02 21 PM" src="https://user-images.githubusercontent.com/2124984/56510214-227fa080-64f7-11e9-8ad7-0dee910a1dd7.png">

#### Testing instructions

* Switch to this PR
* Create a new site from `/start`
* Navigate to Stats, and click through each of the various sections.
* You should not see the overlay above.

Fixes #32395
